app/src/main/java/com/dscvit/handly/onesignal/NotificationReceivedHandler.kt

```diff
@@ -0,0 +1,15 @@
+package com.dscvit.handly.onesignal
+
+import android.util.Log
+import com.onesignal.NotificationExtenderService
+import com.onesignal.OSNotificationReceivedResult
+
+
+class NotificationReceivedHandler: NotificationExtenderService() {
+    override fun onNotificationProcessing(notification: OSNotificationReceivedResult?): Boolean {
+        val data = notification!!.payload.additionalData
+        Log.d("esh", data.toString())
```

Comment:
Small nitpick, but I'm not a fan of the `TAG` here. Maybe use the class' `simpleName`?

Reply:
Sure